### ISSUE TITLE
Exception_INT_Divide_By_Zero from Starcraft.exe

### ISSUE THREAD

**Issue by s7jones** [bwapi/bwapi#776]
I'm playing my bot on the LAN mode against another bot in a VM and I'm starting to get a lot of the following error:

```
//////////////////////////////////////////////////
TIME: Thu Feb 15 14:23:34 2018

VERSION: 1.16.1.1
BWAPI:
  REVISION: 4842
  CLIENT VERSION: 10002
  BUILD: RELEASE
  ERROR: Invalid_Parameter
  LOCATION: Multiplayer 

EXCEPTION: 0xC0000094    EXCEPTION_INT_DIVIDE_BY_ZERO
FAULT:     0x004D933E    StarCraft.exe
REGISTERS:
  EDI: 00000000
  ESI: 00000008
  EBX: 00000001
  EDX: 00000000
  ECX: FFFFFFFC
  EAX: 000003E8
  EBP: 0019FE08
  EIP: 004D933E
  ESP: 0019FDD8
STACK:
  StarCraft.exe     0x004D933E    resetLastInputFrameCounts
  StarCraft.exe     0x004EEDC6    LoadGameCore
  StarCraft.exe     0x004E0720    preLoadGame
  StarCraft.exe     0x004E0B20    WinMain(x,x,x,x)
  StarCraft.exe     0x00404DA5    start
  KERNEL32.DLL      0x76F08654    BaseThreadInitThunk
  ntdll.dll         0x77024A77    RtlGetAppContainerNamedObjectPath
  ntdll.dll         0x77024A47    RtlGetAppContainerNamedObjectPath
```

The stack frame doesn't appear to have anything to do with my bot, although these crashes have suddenly become more frequent. Is there anything I could be doing to increase these crashes?

**Comment by ratiotile** [comment]
Diff your code with the last working version, carefully. I had a strange error in an unrelated .dll because of a misplaced "}". The code might compile but produce undefined behavior that blows up somewhere else.

**Comment by N00byEdge** [comment]
Set the `speed_override` variable to a nonzero value. This was fixed in #741 but the fix is not present in any released versions.

**Comment by N00byEdge** [comment]
Fixed in >4.2.0
